projects/chef-automate.md

```diff
@@ -10,11 +10,19 @@ deployment topologies supported by `chef-automate deploy`.
 ## Project Members
 
 ### Project Owners
+- Vivek Kumar
+  - Github:[vivekvardwaz](https://github.com/vivekvardwaz)
+- Kallol Roy
```

Comment:
People should only be in a single group so if someone is an owner they shouldn't be in groups below that. The GitHub inheritance handles that automatically.